Gemfile.lock

```diff
@@ -124,7 +124,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
be careful not to check in these gemfile changes

Reply:
I saw that. Definitely my bad. I am working on removing it as we speak.